src/Catel.Test/Catel.Test.Shared/GlobalInitialization.cs

```diff
@@ -34,7 +35,9 @@ public static void SetUp()
         modelEqualityComparer.CompareValues = true;
         modelEqualityComparer.CompareCollections = true;
 
-        //System.Threading.Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
+        var culture = new CultureInfo("en-US");
+        System.Threading.Thread.CurrentThread.CurrentCulture = culture;
```

Comment:
CurrentCulture does not affect the tests right now, but I think we should force it anyway.

Reply:
Oh, also I have to mention: with this code it will be impossible to change the culture for the specific test by using SetCultureAttribute. I tried it, and it didn't work.